zilencer/models.py

```diff
@@ -161,13 +161,19 @@ class AbstractRemoteRealmBillingUser(models.Model):
     user_uuid = models.UUIDField()
     email = models.EmailField()
 
+    date_joined = models.DateTimeField(default=timezone_now)
```

Comment:
What's the thinking in having this on the preregistration model? I guess we may want a "when did this happen" timestamp on that, and there's not an obviously better name for that timestamp? Though we do have `confirmation.date_sent` available...

Reply:
@timabbott Hmm, mostly that the regular `PreregistrationUser` has `invited_at`. Though the relationships between `PreregistrationUser`s and `Confirmation` aren't always one-to-one if i recall correctly, so it's less clean than here. But we've also had cases of objects orphaned with the `Confirmation` gone, so I just tend not to trust that model I suppose - and having the extra field seems harmless, while potentially useful.

I considered `date_created` is more accurate as a name, but went with the simpler "let's just use the same name for both PreregBillingUsers and BillingUsers. I can swap it back, if that sounds better

Reply:
This is probably fine; should be easy to change later if we decide to do that.